content/docs/command-reference/repro.md

```diff
@@ -291,7 +296,7 @@ of only the target (`count`) and following stages (none in this case):
 ```dvc
 $ dvc repro --downstream count
 Running stage 'count' with command:
-        python process.py numbers.txt > count.txt
+	python process.py numbers.txt > count.txt
 Updating lock file 'dvc.lock'
```

Comment:
I didn't add "with command" back as it's unnecessary. And, the indent is replaced with ">" in the output.

Reply:
Id if "with command(s)" is unnecessary. Better to be explicit? What if someone just uses DVC for the first time and `pull` + `repro` are their first commands? They don't necessarily realize that a stage contains terminal commands. I say keep it (but make it plural when there's >1 cmd).

Reply:
Oh I guess you've already merged it your way in iterative/dvc/pull/4934 for now. OK